### PR TITLE
Release to Sonatype

### PR DESCRIPTION
**Pull request by bcarter97**
## Description

- Maven Jira request for `uk.sky` access: https://issues.sonatype.org/browse/OSSRH-77287

### CODE REVIEW

**Review by paulaylingdev — APPROVED** [review]

File: build.sbt [hunk 1]
```diff
@@ -4,8 +4,20 @@ lazy val scala213               = "2.13.7"
...
+organization           := "uk.sky"
+sonatypeCredentialHost := "s01.oss.sonatype.org"
+sonatypeRepository     := "https://s01.oss.sonatype.org/service/local"
+homepage               := Some(url("https://github.com/sky-uk/kafka-topic-loader"))
+licenses               := List("BSD New" -> url("https://opensource.org/licenses/BSD-3-Clause"))
+developers             := List(
+  Developer(
+    "BiBCD",
+    "BiBCD",
+    "bibcd-jenkins-github-user@skyglobal.onmicrosoft.com",
```
Comment (lacarvalho91) [inline build.sbt:17]: this feels like it should come from the environment somehow (not sure how that works with Travis though). and i believe it using the BiBCD user is just an implementation detail, so i'm wondering if this should be like:
```
  Developer(
    "Sky UK OSS",
    "Sky UK OSS",
    "SONATYPE_EMAIL",
Attach files by dragging & dropping, selecting or pasting them.
    url("https://github.com/sky-uk/kafka-topic-loader")
  )
```

Comment (bcarter97) [reply build.sbt:17]: Yeah sounds like a good idea, getting it from the environment would give us an option so we can just put the repo url (not an email) as a fallback

Comment (bcarter97) [reply build.sbt]: [`824601a` (#74)](https://github.com/sky-uk/kafka-topic-loader/pull/74/commits/824601a84f321bbd7b6f7982316bab2eef9764e8)

File: build.sbt [hunk 2]
```diff
@@ -4,8 +4,20 @@ lazy val scala213               = "2.13.7"
 lazy val scala212               = "2.12.15"
 lazy val supportedScalaVersions = List(scala213, scala212)
 
-organization := "com.sky"
-name         := "kafka-topic-loader"
+name                   := "kafka-topic-loader"
+organization           := "uk.sky"
+sonatypeCredentialHost := "s01.oss.sonatype.org"
+sonatypeRepository     := "https://s01.oss.sonatype.org/service/local"
+homepage               := Some(url("https://github.com/sky-uk/kafka-topic-loader"))
```
Comment (lacarvalho91) [inline build.sbt:11]: lets put `"https://github.com/sky-uk/kafka-topic-loader"` in a `val` and reuse

Comment (bcarter97) [reply build.sbt]: [`a410eb7` (#74)](https://github.com/sky-uk/kafka-topic-loader/pull/74/commits/a410eb7a94e454123e2aa6988695ca0415a570a7)

**Review by lacarvalho91 — APPROVED** [review]
LGTM, I'm pairing with Paolo soon though so I'll make sure hes OK with it before we merge.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,66 +1,57 @@
...
-1. Follow the [sbt-bintray publishing guidelines](https://github.com/sbt/sbt-bintray#publishing) to ensure you are
-   authenticated with Bintray.  Note that you need to be a member of the `sky-uk` Bintray organization. 
-2. Run `sbt release` to perform the release of the binaries to Bintray.
-3. Check you are happy with the draft publication of the new version [here](https://bintray.com/sky-uk/oss-maven/kafka-topic-loader)
-   and, if so, run `sbt bintrayRelease` to make the new version publicly accessible.
+Merging a PR to master will create a snapshot release in the [Sonatype snapshot repository](https://s01.oss.sonatype.org/content/repositories/snapshots/uk/sky/).
+
+To create a new stable release, you must tag the head of master and push it to Github.
+
+```bash
```
Comment (lacarvalho91) [inline CONTRIBUTING.md:25]: instead of this, we should create a GitHub release instead (which will create the tag you specify on the UI)

Comment (paoloambrosio-skyuk) [reply CONTRIBUTING.md:25]: Creating a tag is a manual task anyway, so I'm happy to use the UI to create a release (and have that create the tag).

**Review by paoloambrosio-skyuk — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (bcarter97) [reply CONTRIBUTING.md:25]: Yeah either works, as long as the tag is created on the remote the job will run

Comment (bcarter97) [reply CONTRIBUTING.md]: [`fb94c8b` (#74)](https://github.com/sky-uk/kafka-topic-loader/pull/74/commits/fb94c8bb11f80f9876f4613f4be419ac1696f7e2)

**Review by lacarvalho91 — APPROVED** [review]

**Pull request merged**